app/_locales/en/messages.json

```diff
@@ -1130,7 +1130,7 @@
     "message": "import using Secret Recovery Phrase"
   },
   "importAccountMsg": {
-    "message": " Imported accounts will not be associated with your originally created MetaMask account Secret Recovery Phrase. Learn more about imported accounts "
+    "message": "Imported accounts will not be associated with your originally created MetaMask account Secret Recovery Phrase. Learn more about imported accounts "
```

Comment:
Do we need the space at the end?

Reply:
The leading space was not needed, but looks like the space at the end serves as the padding between the text and the "here" link

I've removed the space and fixed via styling
<img width="207" alt="Screen Shot 2021-11-10 at 9 16 23 AM" src="https://user-images.githubusercontent.com/8732757/141151065-0052dadf-5ebb-40b1-b649-fcd863cafa56.png">
: